relay-server/src/metrics.rs

```diff
@@ -370,6 +370,16 @@ pub enum RelayCounters {
     ConnectorErrors,
     /// Number of upstream connections that experienced a timeout.
     ConnectorTimeouts,
+    /// Counts the number of upstream http request.
+    ///
+    /// It has the following tags:
```

Comment:
_nit_: To keep this consistent with the other metrics:

```suggestion
    /// This metric is tagged with: 
    ///
```

There are more tags: `status_code`, `route` and `retries`.